src/web/modules/payouts/reconcile.ts

```diff
@@ -0,0 +1,167 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-restricted-syntax */
+import * as Stripe from 'stripe'
+import * as HTTPSProxyAgent from 'https-proxy-agent'
+import * as _ from 'lodash'
+
+import * as config from '../../../config'
+import { renderCSV, PayTransactionCSVEntity, PaymentType } from './csv'
+import { Connector } from '../../../lib/pay-request'
+
+const stripe = new Stripe(process.env.STRIPE_ACCOUNT_API_KEY)
+stripe.setApiVersion('2018-09-24')
+
+// @ts-ignore
+if (config.server.HTTPS_PROXY) stripe.setHttpAgent(new HTTPSProxyAgent(config.server.HTTPS_PROXY))
+
+const getTransactionsForPayout = async function getTransactionsForPayout(
+  stripeAccountId: string,
+  payout: Stripe.payouts.IPayout
+): Promise<Stripe.IList<Stripe.balance.IBalanceTransaction>> {
+  const options: Stripe.balance.IBalanceListOptions = {
+    limit: 100,
```

Comment:
It's been discussed. If there are more than 100 transactions for the payout the verification step won't succeed and the csv won't be generated.

In the future the pagination would need to be supported.

Reply:
Yes! Some thoughts about how this util could work: https://github.com/sfount/stripe-utils/blob/master/lib/pages.js

To confirm payouts will fail if there are more than 100 as it won't match the payout total -- we can work on this if this starts to fail for support